### PR TITLE
Few adjustments in SlackBuild

### PR DESCRIPTION
**Pull request by pm-kan**
* fixed typo in `if` condition
* fixed build from path with spaces
* move man and docs into /usr instead of /usr/share

### CODE REVIEW

**Comment by ronys** [comment]
Will merge this once I release Fedora and Ubuntu packages of 1.05, just to keep things simple.

File: install/slackware/passwordsafe.SlackBuild [hunk 1]
```diff
@@ -103,69 +107,75 @@ PKGNAME=${PRGNAM}-${VERSION/-/.}-${ARCH}-${BUILD_ID}${BUILDER_ID}
...
 
 # collect files before packaging
 # copy docs
-mkdir -p ${PKG}/${DOCDIR}
-cp ${GIT_ROOT}/README* ${GIT_ROOT}/LICENSE ${GIT_ROOT}/install/copyright ${PKG}/${DOCDIR}
-if [-d "${PKG}${DEFDOCDIR}" ]; then
-    mv ${PKG}${DEFDOCDIR}/* ${PKG}/${DOCDIR} || true
-    rmdir ${PKG}/${DEFDOCDIR}
+mkdir -p "${PKG_TMP}${DOCDIR}"
+cp "${GIT_ROOT}"/README* "${GIT_ROOT}"/LICENSE "${GIT_ROOT}"/install/copyright "${PKG_TMP}${DOCDIR}"
```
Comment (dskrvk) [inline install/slackware/passwordsafe.SlackBuild]: Missing / in `"${PKG_TMP}${DOCDIR}"`?

Comment (dskrvk) [reply install/slackware/passwordsafe.SlackBuild]: Same below. Did you intend for `PKG_TMP` to end in a `/`?

Comment (pm-kan) [reply install/slackware/passwordsafe.SlackBuild]: No, (DEF)DOCDIR, (DEF)MANDIR, ... already have leading slash because prefix is `/usr`

**Pull request merged**